### PR TITLE
Updates for Media features and Subresource Integrity

### PR DESCRIPTION
**Pull request by kypflug**
* Subresource Integrity now available behind the Service Worker flag (16257+)
* Screen Capture API is In Development
* Added new backlog features for tracking: Media Capture from Canvas, Media Capture from HTML Media Element, and Audio Output Devices API

### CODE REVIEW

File: status.json [hunk 1]
```diff
@@ -4626,6 +4626,18 @@
     "id": 5522768674160640
   },
   {
+    "name": "Audio Output Devices API",
+    "category": "Multimedia",
+    "link": "https://w3c.github.io/mediacapture-fromelement",
+    "summary": "The Audio Output Devices API allows a Web application to direct audio output to authorized endpoints other than the user agent default.",
+    "standardStatus": "Editor's Draft",
```
Comment (alrra) [inline status.json]: Needs to be `draft`, not `Draft`.

File: status.json [hunk 2]
```diff
@@ -2342,7 +2342,9 @@
     "summary": "Subresource Integrity defines a mechanism by which user agents may verify that a fetched resource has been delivered without unexpected manipulation. In a nutshell, metadata inlined into HTML elements allows the browser to determine whether the resource that was downloaded matches the resource the page's author expected to download.",
     "standardStatus": "Working draft or equivalent",
     "ieStatus": {
-      "text": "Under Consideration"
+      "text": "Preview Release",
+      "ie": "16257",
```
Comment (alrra) [inline status.json]: Needs to be either `iePrefixed` or `ieUnprefixed`.

**Review by alrra — CHANGES_REQUESTED** [review]

**Comment by alrra** [comment]
@kypflug Build [is failing](https://travis-ci.org/MicrosoftEdge/Status/builds/260416830#L468). Left some inline comments that need to be addressed.

**Comment by alrra** [comment]
Merged with some modifications (i.e. integrated the fixes in the correct commits, and changed the commit messages a bit).

**Comment by mkurz** [comment]
The uservoice ticket for subresource integrity hasn't been update yet:
https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/6263699-subresource-integrity

**Comment by molant** [comment]
@dstorey can you update it?

**Pull request closed without merging**